lib/cleanup/destroyable_records.rb

```diff
@@ -58,15 +58,14 @@ def print_data
   end
 
   def destroy_records
-    stdout.puts 'Destroying integration usages'
-    integration_usages.each do |integration_usage|
-      integration_usage.destroy!
+    if integration
```

Comment:
small nitpick but i prefer 
```suggestion
    if integration.present?
```
as it's a bit more explicit